src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -45,20 +40,29 @@
          auth-handler
          puppet-version)
        {:normalize-request-uri true})
-     (when (not= ca-crl-file host-crl-file)
-       (watch-protocol/add-watch-dir! watcher
-                                      (fs/parent ca-crl-file)
-                                      {:recursive true})
-       (watch-protocol/add-callback!
-        watcher
-        (fn [events]
-          (when (some #(and (:changed-path %)
-                            (= (.getCanonicalPath (:changed-path %))
-                               ca-crl-file))
-                      events)
-            (ca/retrieve-ca-crl! ca-crl-file host-crl-file)))))
-     (assoc context :auth-handler auth-handler
-                    :watcher watcher)))
+       (if-let [filesystem-watch-service
```

Comment:
actually the whitespace is off on this whole function because of the single space indent after the args